plugins/templates/list.js

```diff
@@ -4,6 +4,9 @@ const boom = require('boom');
 const joi = require('joi');
 const schema = require('screwdriver-data-schema');
 const listSchema = joi.array().items(schema.models.template.get).label('List of templates');
+const distinctSchema = joi.string().label('Field to return unique results by');
```

Comment:
this can be `joi.string().valid(Object.keys(schema.models.template.base))` so that it's always valid field